_posts/2025-2-27-wdl-survery.markdown

```diff
@@ -0,0 +1,75 @@
+---
+title: "WDL Survey Report"
+author: Venkat Malladi
+date:   2025-02-27 15:24:15 -0600
+categories: wdl bioinformatics workflows 
+---
+
+# WDL Usage and Adoption Survey Report  
+
+# WDL Usage and Adoption Survey Report
```

Comment:
This is copied twice.

Reply:
Honestly, I don't think either h1 is needed here. The title is right above it.

Reply:
good catch